codegen/src/main/java/software/amazon/awssdk/codegen/emitters/CodeWriter.java

```diff
@@ -72,7 +75,7 @@ public CodeWriter(String dir, String file) {
      * @param fileNameSuffix
      *             suffix to be appended at the end of file name.
      */
-    public CodeWriter(String dir, String file, String fileNameSuffix) {
+    public CodeWriter(String dir, String file, String fileNameSuffix, boolean disableFormatter) {
```

Comment:
I had to add this this because the formatting time explodes on on the really big rules (e.g. S3).